File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>
void print_row(int spaces, int bricks);
int get_hight(void;)
int main(void)
{
    // prompt the user for pyrmid's hight
    int n = get_hight();

    //printing pyrmid of hight
    for (int i = 0; i < n; i++)
    {
        // print row of bricks and spaces
        print_row(n - i, i + 1);
    }

}
void print_row(int spaces , int bricks)
{
    // print spaces
    for (int j = 1; j < spaces; j++)
    {
        printf(" ");
    }
    // print bricks
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}
int get_hight(void)
{
    int n;
    do
    {
        n = get_int("Hig)
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height <= 8 and height >= 1:
        break
n = 1
for i in range(height):
    for k in range(height - n):
        print(" ", end="")
    for j in range(n):
        print("#", end="")
    print("")
    n += 1
